src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingSifter.java

```diff
@@ -84,10 +71,10 @@ public BuildingSifter(final IColony c, final BlockPos l)
     {
         super(c, l);
 
-        if (!IColonyManager.getInstance().getCompatibilityManager().getMeshes().isEmpty())
-        {
-            this.sifterMesh = IColonyManager.getInstance().getCompatibilityManager().getMeshes().get(0);
-        }
+        keepX.put(stack -> ItemStackUtils.compareItemStacksIgnoreStackSize(stack, new ItemStack(ModItems.sifterMeshString), false, true), new net.minecraft.util.Tuple<>(1, false));
+        keepX.put(stack -> ItemStackUtils.compareItemStacksIgnoreStackSize(stack, new ItemStack(ModItems.sifterMeshFlint), false, true), new net.minecraft.util.Tuple<>(1, false));
+        keepX.put(stack -> ItemStackUtils.compareItemStacksIgnoreStackSize(stack, new ItemStack(ModItems.sifterMeshIron), false, true), new net.minecraft.util.Tuple<>(1, false));
+        keepX.put(stack -> ItemStackUtils.compareItemStacksIgnoreStackSize(stack, new ItemStack(ModItems.sifterMeshDiamond), false, true), new net.minecraft.util.Tuple<>(1, false));
     }
```

Comment:
Can we create like a tag and just keep 4 meshes independently of the type or so?